pyodide-build/pyodide_build/mkpkg.py

```diff
@@ -73,8 +84,35 @@ def make_package(package: str, version: Optional[str] = None):
         yaml.dump(yaml_content, fd, default_flow_style=False)
 
 
-def update_package(package: str):
-    import yaml
+class bcolors:
+    HEADER = "\033[95m"
+    OKBLUE = "\033[94m"
+    OKCYAN = "\033[96m"
+    OKGREEN = "\033[92m"
+    WARNING = "\033[93m"
+    FAIL = "\033[91m"
+    ENDC = "\033[0m"
+    BOLD = "\033[1m"
+    UNDERLINE = "\033[4m"
+
+
+def abort(msg):
+    print(bcolors.FAIL + msg + bcolors.ENDC)
+    sys.exit(1)
+
+
+def warn(msg):
+    warnings.warn(bcolors.WARNING + msg + bcolors.ENDC)
```

Comment:
This is reinventing a logging library a bit. It doesn't necessarily make sense to define these utils functions here and not somewhere where they could be used by other scripts. As then one need to check that this renders OK on various terminals which is not certain.

If we want to go there, maybe we should consider either [loguru](https://github.com/Delgan/loguru#pretty-logging-with-colors) or switch this CLI to typer (example of using colors [here](https://typer.tiangolo.com/tutorial/printing/), or some other appropriate library. Something that could be discussed in a separate PR.

Though for this PR it's fine.